publications/abstracts/reservoir-computing/results.tex

```diff
@@ -0,0 +1,26 @@
+\section{Results}
+
+Current results include the generation of time series data to be used in
+training the RC model.
+
+\begin{figure}[h]
+ 	\centering
+ 	\label{fig:grid-demand}
+ 	\includegraphics[width=0.8\columnwidth]{syn_typ_demand.png}
+ 	\caption{The typical year of hourly grid demand in kW at UIUC.}
```

Comment:
```suggestion
 	\caption{A typical year of hourly grid demand in kW at UIUC.}
```

Reply:
(you used "kWh per hour" below, but hourly demand in kW here.... perhaps  choose one for consistency)